address issues in DOC-3935 (#411)
redis/docs: content/develop/interact/search-and-query/indexing/_index.md

## Index JSON arrays as TAG

> The preferred method for indexing a JSON field with multivalued terms is through use of JSON arrays.  Each value of the array is indexed.  Those values should be scalars.  If you want to index string or boolean values as TAG within a JSON array, use the [JSONPath]({{< relref "/develop/data-types/json/path" >}}) wildcard operator.

Reviewer: ```suggestion
The preferred method for indexing a JSON field with multivalued terms is using JSON arrays. Each value of the array is indexed, and those values must be scalars. If you want to index string or boolean values as TAGs within a JSON array, use the [JSONPath]({{< relref "/develop/data-types/json/path" >}}) wildcard operator.
```
